### DISCUSSION TITLE
Cannot render and save converted file

### DISCUSSION THREAD

**Discussion by annmoser** [lagmoellertim/unsilence#66] [Q&A]
Hi!

I found this package and would love to use it for my music library. Unfortunately, rendering gives me the following error messages:

FileNotFoundError: [WinError 2] The System cannot find the file: '<path>\\.tmp\\2751f292-3a78-4d7c-a0e1-6de5844f1c09\\out_final.mp3' -> '<path>\\test.mp3'                                                                                                                                                    During handling of the above exception, another exception occurred:                                                                                                                                                                             FileNotFoundError: [Errno 2] No such file or directory: '<path>\\2751f292-3a78-4d7c-a0e1-6de5844f1c09\\out_final.mp3'

The folder .tmp and the randomly named subfolder is created. However, there is only a file "concat_list.txt" in this folder. Do you have an idea what could be wrong?

Thanks so much for your help and Cheers from Switzerland :)

**Comment by lagmoellertim** [comment]
Hey, are you able to send me the file (or have any other file where the same error comes up)?
This seems to be a ffmpeg conversion error and in order to diagnose and fix it, a file is needed for me to debug.

**Reply by annmoser** [reply]
Hi! I'm attaching the .py file which is analoguous to the demo script, along with the kernel output that I got. When I tried installing ffmpeg directly from the console, it only found the 4.2.2 version. So I made a new environment and installed the ffmpeg-snapshot.tar.bz2 from the ffmpeg-website. This is the 4.3.1 version. I get the error with both ffmpeg-versions.

Thanks for the help!

[render issues.zip](https://github.com/lagmoellertim/unsilence/files/5824437/render.issues.zip)

**Reply by Rohan-IITG** [reply]
Hi, I am facing same problem while using WAV file. Have you found any solution?
